docs/action-versioning.md

```diff
@@ -42,7 +42,7 @@ Binding to the immutable full sha1 may offer more reliability.  However, note th
 
 ```
 git tag -fa v1 -m "Update v1 tag"
-git push origin v1 --force
+git push origin v1 --tags --force
```

Comment:
This pushes all tags, which may be a weird side effect that people aren't expecting. Is there a reason you want to include it here? The `v1` refspec should resolve to the tag and push it